composer/trainer/trainer.py

```diff
@@ -838,12 +838,18 @@ def __init__(
         # Console Logging
         loggers = list(ensure_tuple(loggers))
         if any(isinstance(x, ProgressBarLogger) for x in loggers):
-            warnings.warn(
-                DeprecationWarning(
-                    (f'Specifying the {ProgressBarLogger.__name__} via `loggers` is deprecated. Instead, '
-                     'please specify `progress_bar`, `log_to_console`, `log_level`, and `stream` arguments when '
-                     'constructing the trainer. If specified, these arguments will be ignored, as the '
-                     f'{ProgressBarLogger.__name__} was already created.')))
+            if log_to_console is not None:
+                raise ValueError(
```

Comment:
Since we only have one `ProgressBarLogger` instance now, I don't think we need these changes anymore...

Reply:
I think we still do right? Aren't these for the default progress bar? Won't the user might want to construct the `dataloader_label_to_metrics` dict, which gets passed into the logger object. Should I make the `dataloader_label_to_metrics` a trainer parameter instead?

Reply:
Yes, let's also propagate this parameter through the trainer and trainer hparams.